torch/_dynamo/variables/torch.py

```diff
@@ -379,8 +408,7 @@ def get_state_from_generator():
                 for x in args[0].items:
                     if isinstance(x.value, numpy.generic):
                         x.value = x.value.item()
```

Comment:
Right here, rather than passing self.value directly, compute value and override it as necessary?